src/main/kotlin/fi/metatavu/edufication/pages/api/persistence/model/ContentBlock.kt

```diff
@@ -0,0 +1,32 @@
+package fi.metatavu.edufication.pages.api.persistence.model
+
+import fi.metatavu.edufication.pages.api.model.ContentBlockLayout
+import java.time.OffsetDateTime
+import java.util.*
+import javax.persistence.*
+
+@Entity
+class ContentBlock {
```

Comment:
This entity does not track modification details (creator, modifier, create time, modify time) is this intentional?

Reply:
Yes, there is no separate ContentBlock api so the tracking info will be stored on the Page object when ever content is modified.